app/0.13.x/admin-api.md

```diff
@@ -55,6 +55,7 @@ route_body: |
     `methods`<br>*semi-optional*   | A list of HTTP methods that match this Route. For example: `["GET", "POST"]`. At least one of `hosts`, `paths`, or `methods` must be set. With form-encoded, the notation is `methods[]=GET&methods[]=OPTIONS`. With JSON, use an Array.
     `hosts`<br>*semi-optional*     | A list of domain names that match this Route. For example: `example.com`. At least one of `hosts`, `paths`, or `methods` must be set. With form-encoded, the notation is `hosts[]=foo.com&hosts[]=bar.com`. With JSON, use an Array.
     `paths`<br>*semi-optional*     | A list of paths that match this Route. For example: `/my-path`. At least one of `hosts`, `paths`, or `methods` must be set. With form-encoded, the notation is `paths[]=/foo&paths[]=/bar`. With JSON, use an Array.
+    `regex_priority`<br>*optional* | When evaluating the regex path of a Route, the `regex_priority` of the Route will be considered. Determines the relative order in which incoming requests are evaluated against defined Routes. Defaults to `0`.
```

Comment:
Thanks for the feedback! I am aware that this option is documented in the proxy guide. It's just that it was thought that it would be good to have this information documented in the admin API as well for completeness. I have managed to replace the description with your leaner version.